### ISSUE TITLE
Add 'Calculated Gender' to identities

### ISSUE THREAD

**Issue by hmitsch** [mozilla/participation-metrics-org#132]
* Gender is matched based on names
* API used: https://genderize.io
* Analysis based on statistical numbers, not individuals' data

Similar to https://github.com/mozilla/participation-metrics-org/issues/80

**Comment by emmairwin** [comment]
Hi Henrik, Can you tell me more about this?

**Comment by hmitsch** [comment]
Yes! I'd like to discuss with Bitergia if it is possible to add "gender calculation" (via the genderize.io API) to the GrimoireLab identity heuristics (I think they call that Sorting Hat).

I am aware that this is not perfect and that it will be wrong for some identities. However, this is currently the best idea that comes into my mind.

Any other suggestions?

**Comment by emmairwin** [comment]
Do you know how it handles non-english names?  Or I think that's an important question to ask. 
Long-term we need standards for gender identity that make this easier ( I know you know that ).

**Comment by hmitsch** [comment]
Unfortunately I do not know how genderize works on non-English names. I know that this is not the first time that Bitergia is facing such a question and want to rely on their expertise and advice.

Previous work from Bitergia that might be helpful here:
- Diversity in Open Source panel at Linux Foundation Open Source Summit
https://speakerdeck.com/bitergia/diversity-in-open-source-panel

- Gender-diversity Analysis of the Linux Kernel Technical Contributions
https://blog.bitergia.com/2016/10/11/gender-diversity-analysis-of-the-linux-kernel-technical-contributions/
https://speakerdeck.com/bitergia/gender-diversity-analysis-of-the-linux-kernel-technical-contributions

- Gender Diversity Analysis of the Technical Contributions in the Hadoop ecosystem
https://speakerdeck.com/bitergia/gender-diversity-analysis-of-technical-contributions-in-the-hadoop-ecosystem

- Gender Diversity Analysis of the Technical Contributions to OpenStack (Barcelona Summit)
https://speakerdeck.com/bitergia/gender-diversity-analysis-of-technical-contributions-in-openstack-barcelona-edition
https://es.pinterest.com/pin/350084571018915431/

- OpenStack gender/diversity technical contributions analysis (Austin Summit)
https://blog.bitergia.com/2016/04/25/openstack-genderdiversity-technical-contributions-analysis/
https://es.pinterest.com/pin/350084571016264818/

**Comment by emmairwin** [comment]
Thanks Henrik going to learn more from Daniel  on Monday as well.  Its relevant to goals for pushing standards for D&I so we can move forward from gender binary solutions.   Thanks!
